doc/Extensions/Applications.md

```diff
@@ -715,3 +716,21 @@ extend ups-apcups /etc/snmp/ups-apcups.sh
 4. Restart snmpd on your host
 
 5. On the device page in Librenms, edit your host and check the `UPS apcups` under the Applications tab.
+
+
+### SDFS info
+A small shell script that exportfs SDFS volume info.
+
+###### SNMP Extend
+1. Copy the [sdfsinfo](https://github.com/librenms/librenms-agent/blob/master/snmp/sdfsinfo) to `/etc/snmp/` on SDFS enabled host.
```

Comment:
Last comment from me I think, you should put the curl command like in the other instructions so people can just copy / paste.